### PR TITLE
feat : added wishlist management

### PR DESCRIPTION
**Pull request by someshtiwarii**
- wishlist page (demo cards added)

### CODE REVIEW

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import url("/css/general.css");
+
+.wishlist-items {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: repeat(2, auto);
```
Comment (msdeshmukh009) [inline pages/wishlist/wishlist.css:6]: I might be wrong here but I think even if you do not define the rows explicitly the browser will take care of it. So maybe try removing this line.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: pages/wishlist/wishlist.css (hunk 1 above)
Comment (someshtiwarii) [reply pages/wishlist/wishlist.css:6]: yes youre right Mahesh :) I will do it

**Review by manojasarsa — APPROVED** [review]
Good to merge!

**Comment by preethasri** [comment]
good work somesh👍

**Pull request merged**